test/gtest/ucp/ucp_test.cc

```diff
@@ -641,8 +641,11 @@ ucp_ep_h ucp_test_base::entity::accept(ucp_worker_h worker,
         UCS_TEST_SKIP_R("Skipping due an unreachable destination (unsupported "
                         "feature or no supported transport to send partial "
                         "worker address)");
+    } else if (status != UCS_OK) {
+        add_err(status);
+        ep = NULL;
```

Comment:
add_err usually counts number of times the error handler callback was invoked, not number of times some API func failed
IMO it should be some other counter such as number of connection requests dropped/failed

Reply:
done
added `m_err_accept_cntr`